vision/conv_mnist/Project.toml

```diff
@@ -1,14 +1,11 @@
 [deps]
 BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
-Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 MLDatasets = "eb30cadb-4394-5ae3-aed4-317e484a6458"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 TensorBoardLogger = "899adc3e-224a-11e9-021f-63837185c80f"
 
 [compat]
-CUDA = "2.4.0"
-Flux = "0.13"
```

Comment:
These compat bounds should be updated instead of removed. Flux 0.13 looks fine, so I think CUDA 3 should work too.

Reply:
compat bounds now added back and updated